### PR TITLE
Filter values autocomplete rebased master

### PR DESCRIPTION
**Pull request by Gauravshah**
gives autocomplete on filter values by performing a topNQuery.
TopNQuery in druid is very quick and this will help dashboard editor create dashboards without making mistakes.

### CODE REVIEW

**Comment by Gauravshah** [comment]
should I add any screenshots or more details ?

**Comment by abhisant** [comment]
@Gauravshah : Sorry, didn't get a chance to review this. Will do it soon. Thanks for your contributions.

File: src/query_ctrl.ts [hunk 1]
```diff
@@ -127,6 +128,14 @@ export class DruidQueryCtrl extends QueryCtrl {
         .then(callback);
     };
 
+    this.getFilterValues = (query, callback) => {
+      let dimension = this.target.currentFilter.dimension;
+      this.datasource.getFilterValues(this.target, this.panelCtrl.range, query)
```
Comment (Gauravshah) [inline src/query_ctrl.ts]: not sure if we should use last 24 hours or the date range from panel to fulfill the request. As of not using panel time range

**Comment by andresgomezfrr** [comment]
it's a good feature! 👍

**Comment by kvsurii** [comment]
Looking forward to this merge

**Comment by pkarmaka** [comment]
@Gauravshah I don't see the dataSource parameter in the druid query when I run it

**Comment by Gauravshah** [comment]
@pkarmaka can you send me a screenshot of your request & config ? 
https://github.com/GoshPosh/druidplugin/blob/6280ec6acb07fd9e92db0db5f5a58b7cca1bdf3e/dist/datasource.js#L294 should be doing on its own

**Comment by kvsurii** [comment]
@pkarmaka @Gauravshah please see https://github.com/grafana-druid-plugin/druidplugin/issues/34

**Pull request merged**